### ISSUE TITLE
Explain command fails with wildcard in fetchplan

### ISSUE THREAD

**Issue by thierry-github** [orientechnologies/orientdb#5689]
working with 2.1.9 release.

when i run this command, i have an issue :
```
explain SELECT FROM Def fetchplan *:2 [*]in_*:-2
```

```
com.orientechnologies.orient.core.sql.OCommandSQLParsingException: Error on parsing command at position #0: Error parsing query: 
SELECT FROM Def fetchplan *:2 [*]in_*:-2
Encountered " "*" "* "" at line 1, column 32.
Was expecting one of:
    <INTEGER_LITERAL> ...
    "-" ...
    "-" ...
    <INTEGER_LITERAL> ...
```
the query (without explain) works fine.

the problem is : `[*]in_*:-2`

if i change [*]in_*:-2 by [**1**]in_*:-2 it works as expected

**Comment by luigidellaquila** [comment]
Hi @thierry-github 

I'm on it

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @thierry-github 

I just pushed a fix on 2.1.x branch, now I'm porting the fix to develop

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Done, closing. The fix will be released with 2.1.11
Please reopen if you see any other problems

Thanks!

Luigi
